### DISCUSSION TITLE
"Not supported for: repository"... What isn't supported?

### DISCUSSION THREAD

**Discussion by Andre601** [lowlighter/metrics#846] [Q&A]
When using the action in a repository for some SVG stuff (see #835 for context) do I get `Not supported for: repository` warnings before the attempt fails.
Yet, the action doesn't do any attempts at explaining me WHAT isn't working. It just says something is not supported and fails.

Failed Workflow run: https://github.com/Readme-Workflows/recent-activity/runs/5086961166?check_suite_focus=true

Configuration: https://github.com/Readme-Workflows/recent-activity/blob/main/.github/workflows/update-readme.yml

**Comment by lowlighter** [comment]
Seems that the `metadata.yml` of the `markdown` template does not accept repository:
https://github.com/lowlighter/metrics/blob/95290778e6cc3d8483cb0f626909d7091d564f58/source/templates/markdown/metadata.yml#L6-L8

But it's probably just an oversight, I think you're the first using the markdown feature for repository 😅 

Just adding the `- repository` to the `supports` array should solve this issue

**Comment by Andre601** [comment]
Quick update!

I updated the action to v3.20 and while the previous error doesn't seem to appear anymore, is there a new error:
```bash
Error: ejs:81
    79| ### Stargazers
    80|
 >> 81| [{%- await embed(`stats-stargazers`, {stargazers: true, stargazers_charts_type: "chartist"}) %}][stargazers]
    82|
    83| ### Contributors
    84|

template not supported for: repository
```

So it seems like some templates are still not supported for Repositories?

Workflow Run: https://github.com/Readme-Workflows/recent-activity/runs/5123692499?check_suite_focus=true
Latest workflow file: https://github.com/Readme-Workflows/recent-activity/blob/ce08e7bf8e0068eddacb3af0c87fb5726829220f/.github/workflows/update-readme.yml

**Reply by lowlighter** [reply]
I think I found the root cause, it's because the `embed` always default to `template: classic` which indeed is not supported by repository

As a workaround for now you can add `template: "repository", repo: "Readme-Workflows"` to the second argument each time you call `embed()` to force the use of the repository template and it should work from what I tested

**Reply by Andre601** [reply]
*Should* the `repo` be Readme-Workflows and not recent-activity?

**Reply by lowlighter** [reply]
My bad, yes it should be set to recent-activity indeed

**Reply by lowlighter** [reply]
Should have been fixed by #847 and #865
